app/src/main/java/com/developersbreach/composeactors/ui/screens/favorites/tabs/FavoriteActorsTabContent.kt

```diff
@@ -1,19 +1,168 @@
 package com.developersbreach.composeactors.ui.screens.favorites.tabs
 
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.pager.VerticalPager
+import androidx.compose.foundation.pager.rememberPagerState
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.graphics.RectangleShape
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.tooling.preview.Preview
-import com.developersbreach.composeactors.data.model.Actor
+import androidx.compose.ui.unit.dp
+import com.developersbreach.composeactors.R
+import com.developersbreach.composeactors.data.model.FavoriteActor
+import com.developersbreach.composeactors.ui.animations.borderRevealAnimation
+import com.developersbreach.composeactors.ui.components.LoadNetworkImage
 import com.developersbreach.composeactors.ui.screens.favorites.NoFavoritesFoundUI
 import com.developersbreach.composeactors.ui.theme.ComposeActorsTheme
+import com.developersbreach.composeactors.utils.getPlaceOfBirth
 
+@OptIn(ExperimentalFoundationApi::class)
 @Composable
 fun FavoriteActorsTabContent(
     getSelectedActorDetails: (Int) -> Unit,
-    favoriteActors: List<Actor>
+    favoriteActors: List<FavoriteActor>,
+    removeFavoriteActor: (FavoriteActor) -> Unit,
 ) {
     if (favoriteActors.isEmpty()) {
         NoFavoritesFoundUI()
     }
+
+    val listState = rememberPagerState()
+
+    VerticalPager(
```

Comment:
I like this idea of using `VerticalPager`, I'll make some UI changes for this later.